recipe/meta.yaml

```diff
@@ -1,5 +1,13 @@
-{% set build_num = 1 %}
-{% set min_runtime_version = 12 %}
+{% set build_num = 2 %}
+{% set gcc_major = 13 if ctng_gcc is undefined else ctng_gcc.split(".")[0] %}
+# generally, the runtime version of libstdcxx needs to be at least as high
+# as the compiler; however, wherever libstdcxx changes the default ABI version
+# of a symbol, we need to ensure at least that version at runtime also for
+# packages built with older compilers (because that build environment will
+# pull in the newest libstdcxx, and then depend on that new symbol version).
+# C.f. https://johannst.github.io/notes/development/symbolver.html resp.
+# https://github.com/search?q=repo%3Agcc-mirror%2Fgcc%20_GLIBCXX_BEGIN_INLINE_ABI_NAMESPACE&type=code
```

Comment:
This is not relevant.

Reply:
Are you referring to the GH search only, or a larger portion of the comment? If it's the former, would you have anything at hand (e.g. a search pattern, link, procedure etc.) that makes analysing for symbol renames a bit more transparent?

FWIW, I used that search term because it appears directly related to symbol renames, e.g. it gets [used](https://github.com/gcc-mirror/gcc/blob/releases/gcc-13.2.0/libstdc%2B%2B-v3/include/std/condition_variable#L240-L244) as:
```
_GLIBCXX_BEGIN_INLINE_ABI_NAMESPACE(_V2)
```

In fact, as far as I can tell, it was [introduced](https://github.com/gcc-mirror/gcc/blob/releases/gcc-13.2.0/libstdc%2B%2B-v3/ChangeLog-2022#L4934-L4940) exactly for (or at least concurrently with) the symbol version default change of `std::condition_variable` that lead to #75.

Reply:
ABI namespace is not about default versions of symbols. Relevant commit is https://github.com/gcc-mirror/gcc/commit/9e18a25331fa25c3907249fede65a02c6817b06e

Reply:
Thanks!